marklogic-data-hub-central/ui/src/assets/mock-data/modeling.js

```diff
@@ -459,4 +459,215 @@ export const editEntityPropertyRequest = { "AnotherModel": {
     }
   }
 }
-}
\ No newline at end of file
+}
+
+export const propertyTableEntities = [
+  {
+      "entityName": "Concept",
```

Comment:
This is fine, but if possible, I'd like for us to use Customer/Order across the board as defined in reference-entity-model project. We can absolutely enhance those to account for scenarios that they don't account for yet. It keeps us all on the same sheet of music, which makes communicating/testing/debugging easier.